DESCRIPTION

```diff
@@ -18,10 +18,9 @@ Depends:
 Imports:
     httr,
     purrr,
-    tibble
-RoxygenNote: 6.1.1
+    tibble,
+    withr
```

Comment:
New dependency, used in tests. I suppose it could also go in Suggests, alongside testthat.

Reply:
It's ok to add it.